capstone/src/components/User/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import './Header.css';
+
+export default function Header({ setIsLoggedIn }) {
+  function handleLogOut() {
+    localStorage.removeItem('current_user_id');
```

Comment:
Let's say we decide to change the localStorage key (`current_user_id`) - we'll have to change the reference in every file, and if we miss one, things could break. Is there anything we can do to help prevent that? :)